Source/Core/Core/PowerPC/JitCommon/JitBackpatch.cpp

```diff
@@ -71,6 +71,22 @@ bool Jitx86Base::BackPatch(u32 emAddress, SContext* ctx)
 		return false;
 	}
 
+	auto it2 = pcAtLoc.find(codePtr);
+	u32 pc;
+	if (it2 == pcAtLoc.end())
+	{
+		if (info.isMemoryWrite || jit->js.memcheck)
+		{
+			PanicAlert("BackPatch: no pc entry for address %p", codePtr);
```

Comment:
We don't need PC to generate a DSI; the code which actually performs the early exit takes care of that.  (From TrampolineCache.cpp: "PC is used by memory watchpoints (if enabled) or to print accurate PC locations in debug logs".)